test/pkg/environment/common/expectations.go

```diff
@@ -503,11 +504,14 @@ func (env *Environment) ConsistentlyExpectNodeCount(comparator string, count int
 	return lo.ToSlicePtr(nodeList.Items)
 }
 
-// ConsistentlyExpectNoDisruptions ensures that the state of the cluster is not changed within a passed duration
-// Specifically, we check if the cluster size in terms of nodes is the same as the passed-in size and we validate
-// that no disrupting taints are added throughout the window
-func (env *Environment) ConsistentlyExpectNoDisruptions(nodeCount int, duration time.Duration) {
+func (env *Environment) ConsistentlyExpectNoDisruptions(nodeCount int, duration time.Duration) []*v1.Node {
 	GinkgoHelper()
+	return env.ConsistentlyExpectDisruptingNodesWithNodeCount(0, nodeCount, duration)
+}
+
+func (env *Environment) ConsistentlyExpectDisruptingNodesWithNodeCount(nodesTainted int, nodeCount int, duration time.Duration) []*v1.Node {
```

Comment:
nit: It's unclear based on the interface that this is returning back the set of tainted nodes and not all nodes on the cluster